src/Entity/CategoryManager.php

```diff
@@ -146,7 +145,7 @@ public function getBySlug(string $slug, ?string $contextId = null, ?bool $enable
             ->andWhere('c.slug = :slug')->setParameter('slug', $slug);
 
         if (null !== $contextId) {
-            $queryBuilder->andWhere('c.context = :context')->setParameter('context', $contextId, Types::OBJECT);
+            $queryBuilder->andWhere('c.context = :context')->setParameter('context', $contextId);
```

Comment:
This was failing when trying to query with a non mocked CategoryManager on CategoryManagerTest, wdyt? ContextId is not an object, but a string.